src/verified.js

```diff
@@ -6,7 +6,7 @@ class Verified {
    */
   constructor (box) {
     this._box = box
-    this._did = box._3id.getDid()
+    this._did = box._3id.muportDID
```

Comment:
still going to use muport for all verified/claims right now? not just verifying both

Reply:
Yes, it's the simplest thing for now since an ipfs instance is needed to resolve a 3id DID. This would break the `Box.getVerifiedAccounts` method since that doesn't have an ipfs instance.